rancher/rancher
# Adding drainBeforeDelete field to enable the feature in provisioning tests

## Issue: 

 
## Problem

We don't have the option to set the value for drainBeforeDelete in our automation suite. We want to include test cases for this feature and need this option in our framework.
 
## Solution

We are adding the drainBeforeDelete field to allow us to use this feature in our automation suite.
 
## Testing


## Engineering Testing
### Manual Testing

Manual testing was done on local setup to ensure that the field was being added correctly.

### Automated Testing

Jenkins jobs will be available upon request

## QA Testing Considerations

This is for the qa team to be able to run automated tests around the drainBeforeDelete feature
 
### Regressions Considerations

There are no regressions as the feature already exists and we are just adding the field to enable the feature on the automation suite.
